recipe/meta.yaml

```diff
@@ -15,13 +15,14 @@ source:
 build:
     number: 1004
     script: python setup.py install --single-version-externally-managed --record record.txt
+    noarch: python
 
 requirements:
     host:
-        - python
+        - python >=3.7
         - setuptools
     run:
-        - python
+        - python >=3.7
 
 test:
     imports:
```

Comment:
The license below should be

```
license: BSD-3-Clause
```

With a dash and capital as the bot points out.